src/components/elements/upload/fileDialog/FileDialog.tsx

```diff
@@ -9,19 +9,13 @@ import {
   Stack,
```

Comment:
Hm, based on bulletproof-react, I was thinking 
- modules import from components
- components should not import from modules
- modules should avoid importing from other modules

and since a lot of File could is used by both the `clientFiles` and the `form` module, it should be refactored into shared `component` code.

reference https://github.com/alan2207/bulletproof-react/blob/master/docs/project-structure.md
> It might not be a good idea to import across the features. Instead, compose different features at the application level. This way, you can ensure that each feature is independent which makes the codebase less convoluted. ...
> You might also want to enforce unidirectional codebase architecture. This means that the code should flow in one direction, from shared parts of the code to the application (shared -> features -> app). This is a good practice to follow as it makes the codebase more predictable and easier to understand.

However, based on the guidelines in our own readme https://github.com/greenriver/hmis-frontend/pull/948/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 maybe it makes sense to move all file-related code into one shared `files` module as you've suggested above, which `DynamicForm` is still allowed to import from even though it is in the `form` module?
> Prefer to co-locate code that deals with the same type of thing into a shared module, even if the components are rendered in different parts of the application. For example, the Project Services table and Client Services table can live in the same "service" module.
> If two potential features are highly interdependent and frequently change together, consider combining them into a single module.
> (regarding services) because they are relatively independent implementations.

maybe "relatively independent" indicates that they don't have to be totally independent and not import from one another? 

However, is there a reason for us to diverge from bulletproof-react's suggestion here?

Reply:
Dev standup discussion
- Things in modules are domain specific
- Things in src/components are common
- It's ok if this leads to lack of colocation. But they shouldn't be highly interdependent and frequently change together